Assets/Scripts/SethRefactor/ActionLogics/DropOff.cs

```diff
@@ -7,22 +7,23 @@ namespace ScavengerWorld
     [CreateAssetMenu(menuName = "Scavenger World/Action Logics/Drop Off")]
     public class DropOff : ActionLogic
     {
-        public override bool RequiresInRange(Unit agent, Interactable target)
+        public override bool RequiresInRange(Unit unit, Interactable target)
         {
             return true;
         }
 
-        public override void StartAction(Unit agent, Interactable target)
+        public override void StartAction(Unit unit, Interactable target)
         {
-            Debug.Log("Started Dropoff action!");
+            target.Unit.AddItem(unit.RemoveAllItems());
```

Comment:
This works for now, but what would you do if they could separately hold food and an item (e.g., sword)?  We wouldn't want to dump everything in one fell swoop!

Reply:
Yeah this is just a very simple prototype inventory mechanic. When we actually have distinct items, a more complex inventory system will have to be made.